service/configure/updates.go

```diff
@@ -5,6 +5,9 @@ import (
 )
 
 var (
+	DefaultBinaryIndexName = "Portmaster Binaries"
+	DefaultIntelIndexName  = "intel"
```

Comment:
The current name from https://updates.safing.io/intel.v3.json is "Portmaster Intel", in case you wanted to sync with that.